release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -156,6 +156,13 @@ For more information, see _Managing machines with the Cluster API_.
 [id="ocp-4-16-nodes"]
 === Nodes
 
+[id="ocp-4-14-nodes-cgroupv2-default"]
+==== Linux Control Groups version 2 is now generally available (GA)
+
+Beginning with {product-title} 4.16, Control Groups version 2 also known as cgroup v2, cgroup2, or cgroupsv2 is GA. Control Groups version 2 has been the default since 4.14. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the `cgroupMode` field in the `node.config` object to `v1`.
```

Comment:
Is it clear that cgroups v2 is GA for the Real time/Low latency (Telco) use case here ?  
@MarSik  PTAL as well.

Reply:
I was looking at the previous note in 4.14 https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html do I need to explicitly state it is GA for  Real time/Low latency (Telco) use cases here ? @MarSik

Reply:
```suggestion
Beginning with {product-title} 4.16, Control Groups version 2 also known as cgroup v2, cgroup2, or cgroupsv2 is enabled by default for all new deployments even when PerformanceProfiles are present. 

OpenShift has used cgroups v2 by default since 4.14, but the PerformanceProfile functionality required using cgroups v1. This has been resolved now.

cgroup v1 is still used in upgraded clusters with PerformanceProfile(s) that have initial installation dates prior to {product-title} 4.16. cgroup v1 can still be used in the current version by changing the `cgroupMode` field in the `node.config` object to `v1`.
```

I tried to rephrase this a bit to avoid the GA term. The "prior to" should say 4.16, not 4.14.